Source/JavaScriptCore/llint/WebAssembly.asm

```diff
@@ -885,6 +884,262 @@ end
     break
 end)
 
+# This is the interpreted analogue to WasmBinding.cpp:wasmToWasm
+op(wasm_to_wasm_wrapper_entry, macro()
+    loadq Callee-0x10[sp], ws1
```

Comment:
Is `-0x10` the same in x86_64? Maybe it has difference since x86 already pushes return address to the stack?
Maybe it is prologueStackPointerDelta?